CONTRIBUTING.md

```diff
@@ -90,12 +90,70 @@ Follow GitHub's fork-branch-pull request pattern.
 7. Provide your full name or GitHub username and your company or organization if applicable.
 
 #### Once a Pull Request Ready for Review
-1. Verify the commit message and PR title use the template `Fix #XYZ, SHORT_DESCRIPTION`.
-2. Verify there is only one commit message. Squash or amend the commit messages if necessary.
+1. Verify the commit message and PR title use the template `Fix #XYZ, SHORT_DESCRIPTION`. The commit message can use `Fix #XYZ, LONG_DESCRIPTION` if necessary. 
+2. Verify there is one commit message per topic. For example, review the [provided pull request](https://github.com/nasa/cFE/pull/1203/commits). 
+3. Squash or amend the commit messages if necessary. For more information, review the sections [How to Squash Commits](#how-to-squash-commits) and [How to Amend Commits](#how-to-amend-commits) .  
 3. Verify that the PR passes all checks.
-4. The project team will label the issue and evaluate the pull request in the weekly configuration control board (CCB) meeting. For more information, visit [The cFS CCB Process.](https://github.com/nasa/cFS/wiki/The-cFS-CCB-Process)
+4. The project team will label the issue and evaluate the pull request in the weekly configuration control board (CCB) meeting. For more information, visit [The cFS CCB Process](https://github.com/nasa/cFS/wiki/The-cFS-CCB-Process).
 5. If the pull request is accepted, it will be merged into cFS.
 
+#### How to Squash Commits
+1. Switch to the master branch and ensure you are up to date:
+```sh
+git checkout master && git pull
+```
+2. Checkout your feature branch:
+```sh
+git merge feature_branch
+```
+3. Use rebase to open the vi or other editor that lists the commits:
+```sh
+git rebase main -i
+```
+4. A text editor will open with a file that lists all the commits in your branch, and in front of each commit is the word "pick". It looks something like this:
+```sh
+pick 1fc6c95 do something
+pick 6b2481b do something else
+pick dd1475d changed some things
+pick c619268 fixing typos
+```
+5. For every line except the first, you want to replace the word "pick" with the word "squash". It should end up looking like this:
+```sh
```

Comment:
note f skips the step where you have to update the commit message (just uses the previous)

Reply:
Added instructions for both pick and squash as well as r and f.